src/txadmission.cpp

```diff
@@ -790,6 +790,7 @@ bool ParallelAcceptToMemoryPool(Snapshot &ss,
                     if (debugger)
                     {
                         debugger->AddInvalidReason("Inputs are missing");
+                        return false;
```

Comment:
I thought the point of having all these debugger statements was the we would continue and do a full report on every other issue with the transaction, or is that not applicable here for some reason?

Reply:
not applicable here. we will assert false when trying to do mempool entry operations for the tx due to missing data. its an edge case scenario where we cant do tests on the transaction if we dont have any inputs to use.